numpy/linalg/linalg.py

```diff
@@ -17,6 +17,7 @@
 import functools
 import operator
 import warnings
+from typing import NamedTuple
```

Comment:
Curious about this, it isn't the same as collections.namedtuple. Is typing.NamedTuple the compatible version?